src/main/java/io/appform/dropwizard/sharding/metrics/TransactionMetricManager.java

```diff
@@ -0,0 +1,88 @@
+package io.appform.dropwizard.sharding.metrics;
+
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.SlidingTimeWindowArrayReservoir;
+import com.codahale.metrics.Timer;
+import com.google.common.base.Strings;
+import io.appform.dropwizard.sharding.config.MetricConfig;
+import io.appform.dropwizard.sharding.execution.TransactionExecutionContext;
+import lombok.val;
+
+import java.util.concurrent.TimeUnit;
+
+public class TransactionMetricManager {
```

Comment:
Should this be managed so that lifecycle can be managed through container lifecycle ?